src/main/java/nextstep/lms/domain/Student.java

```diff
@@ -0,0 +1,55 @@
+package nextstep.lms.domain;
+
+import nextstep.lms.enums.StudentStatusEnum;
+
+import java.util.Objects;
+
+public class Student {
+    private final Long userId;
+    private final Long sessionId;
+    private StudentStatusEnum studentStatusEnum;
+
+    public Student(Long userId, Long sessionId) {
+        this(userId, sessionId, StudentStatusEnum.APPLIED.name());
+    }
+
+    public Student(Long userId, Long sessionId, String studentStatus) {
```

Comment:
문자열 대신 타입 그대로 주입 받으면 어떨까요?
```suggestion
    public Student(Long userId, Long sessionId, StudentStatusEnum studentStatus) {
```